enterprise/static/enterprise/js/grant_data_sharing_permissions.js

```diff
@@ -47,49 +26,29 @@ function loadConsentPage() {
         $("#data-consent-checkbox").attr("checked", false);
         $("#data-sharing").submit();
     });
-    $("#review-policy-link").click(function (event) {
+    $reviewPolicyLink.click(function (event) {
         event.stopPropagation();
-        hideConsentConfirmationModal();
         togglePolicy(true);
+        $("#consent-confirmation-modal").modal('hide');
         $("#consent-policy-dropdown-bar").focus();
     });
-    $("#consent-confirmation-modal").click(function (event) {
-        hideConsentConfirmationModal();
-        $("#failure-link").focus();
-    });
     $("#data-consent-checkbox").change(function (event) {
         $("#consent-button").attr("disabled", !(this.checked));
     });
-    $(document).keydown(function (event) {
-        if ((event.keyCode === 27) && ($("#consent-confirmation-modal").is(":visible"))) {
```

Comment:
Bootstrap handles this for us now?

Reply:
Yep, Bootstrap 4 handles almost everything JS side for modals as long as you provide the appropriate attributes, which I've made sure to do. You can do extra fancy stuff like catching events but we don't need that right now.